src/build.ts

```diff
@@ -0,0 +1,186 @@
+import { ensureDir, getProtocolInterface, parse, path } from "./deps.ts";
```

Comment:
This code is taken from -builder, and is a combination of [builder's `mod.ts`](https://github.com/slackapi/deno-slack-builder/blob/main/src/mod.ts) and [`functions.ts`](https://github.com/slackapi/deno-slack-builder/blob/main/src/functions.ts).

Some of the logic is different, especially around massaging the file path parameters. In -builder, the code paths for both the `build` and `get-manifest` hooks were intertwined. In this repo, each hook implementation lives in its own file, which IMO simplifies the logic a bit as we don't need conditionals throughout the code to differentiate between the two hook invocations.